lib/white_bread/context.ex

```diff
@@ -6,10 +6,13 @@ defmodule WhiteBread.Context do
   @step_keywords [:given_, :when_, :then_, :and_, :but_]
 
   @doc false
-  defmacro __using__(_opts) do
+  defmacro __using__(opts \\ []) do
+    opts = Keyword.merge [test_library: :ex_unit], opts
```

Comment:
One small change request. Can this be pulled out as `@default_test_library` so that we have 
```elixir
opts = Keyword.merge [test_library: @default_test_library], opts
```
and further up the file:
```elixir
@step_keywords [:given_, :when_, :then_, :and_, :but_]
@default_test_library :ex_unit
```

Reply:
That sounds like a good idea.